index.php

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
```

Comment:
Do you think it's a good idea to put this file here? Won't it change with every commit? 

Maybe creating an empty example / skeleton would be better?